src/Metadata/Resource/Factory/OperationResourceMetadataFactory.php

```diff
@@ -148,4 +156,17 @@ private function normalizeGraphQl(ResourceMetadata $resourceMetadata, array $ope
 
         return $resourceMetadata->withGraphql($operations);
     }
+
+    private function needsDefaultGetOperation(ResourceMetadata $resourceMetadata): bool
+    {
+        $itemOperations = $resourceMetadata->getItemOperations();
+
+        foreach ($itemOperations as $itemOperation) {
+            if ('GET' === ($itemOperation['method'] ?? false)) {
```

Comment:
Why not `?? null` instead?

Reply:
We could document the custom get operation with one that disables it! I'm fixing what you said!

Reply:
> We could document the custom get operation with one that disables it!

What do you mean?

Anyway, I've given it more thought and it's still unsafe anyway. We don't know that there's not already a `GET` item operation because of `route_name`. So I really think we must revert this.

Reply:
see api-platform/docs#886

Reply:
you said it yourself  it's not recommended to use `route_name`?

Reply:
Yes, but if there's an operation using `route_name` which is **the** `GET` item operation, we could never tell, and we would be creating this `get` item operation with `NotFoundAction`. And there's nothing the user could do to fix that.